ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -252,32 +258,28 @@ function Body({ hasReadAccess, isFeatureFlagEnabled }: BodyProps): ReactElement
     return (
         <div className="flex flex-col h-full w-full relative overflow-auto bg-base-100">
             <ErrorBoundary>
-                <Switch>
-                    <Route path="/" exact render={() => <Redirect to={dashboardPath} />} />
-                    <Route path={mainPath} exact render={() => <Redirect to={dashboardPath} />} />
+                <Routes>
+                    <Route path="/" element={<Navigate to={dashboardPath} replace />} />
+                    <Route path={mainPath} element={<Navigate to={dashboardPath} replace />} />
                     {/* Make sure the following Redirect element works after react-router-dom upgrade */}
                     <Route
-                        path={deprecatedPoliciesBasePath}
-                        exact
-                        render={() => <Redirect to={policiesBasePath} />}
+                        path={`${deprecatedPoliciesBasePath}/*`}
+                        element={<Navigate to={policiesBasePath} replace />}
```

Comment:
@alwayshooin Have this and the following redirect exceeded its expiry data?

Although touched in prerequisite contribution by Brad, you added on 2022-06-01 in #1850